documentation/docs/03_account.md

```diff
@@ -9,209 +9,233 @@ It is responsible for the creation and management of account addresses and their
+
 #### Parameters
 
-| Name    | Optional  | Default       | Example |
-| ------- | --------- | ------------- | ------- |
-| `` | ✓ | | |
+| Name      | Optional  | Default | Example                                                           |
+| --------- | --------- | ------- | ----------------------------------------------------------------- |
+| `address` | ✘         | N/A     | "rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3" |
+| `amount`  | ✘         | N/A     | 1                                                                 |
 
-#### Example(s)
+#### Example
 
 ```sh
-> Account "main": send-micro
+> Account "main": send-micro rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3 1
 ```
 
 ### `send-native-token`
 
+Sends native tokens to an address with StorageDepositReturn and Expiration Unlock Condition.
+
 #### Parameters
 
-| Name    | Optional  | Default       | Example |
-| ------- | --------- | ------------- | ------- |
-| `` | ✓ | | |
+| Name        | Optional  | Default | Example                                                                           |
+| ----------- | --------- | ------- | --------------------------------------------------------------------------------- |
+| `address`   | ✘         | N/A     | "rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3"                 |
+| `token_id`  | ✘         | N/A     | "0x08860e1f3593ba86c597cf86f61d8b04d8a714c02c7c5da7132d45be9c2ce6445c0300000000"  |
+| `amount`    | ✘         | N/A     | 100                                                                               |
 
-#### Example(s)
+#### Example
 
 ```sh
-> Account "main": send-native-token
+> Account "main": send-native-token rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3 0x08860e1f3593ba86c597cf86f61d8b04d8a714c02c7c5da7132d45be9c2ce6445c0300000000 100
 ```
 
 ### `send-nft`
 
+Sends a NFT to an address.
+
 #### Parameters
 
-| Name    | Optional  | Default       | Example |
-| ------- | --------- | ------------- | ------- |
-| `` | ✓ | | |
+| Name      | Optional  | Default | Example                                                               |
+| --------- | --------- | ------- | --------------------------------------------------------------------- |
+| `address` | ✘         | N/A     | "rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3"     |
+| `nft_id`  | ✘         | N/A     | "0x397ae8552dcf0dc604a44c9d86a5005d09f95d67e2965ea3b1c1271f9a9ae44c"  |
 
-#### Example(s)
+#### Example
 
 ```sh
-> Account "main": send-nft
+> Account "main": send-nft rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3 0x397ae8552dcf0dc604a44c9d86a5005d09f95d67e2965ea3b1c1271f9a9ae44c
 ```
 
 ### `sync`
 
-#### Parameters
-
-| Name    | Optional  | Default       | Example |
-| ------- | --------- | ------------- | ------- |
-| `` | ✓ | | |
+Synchronises all accounts.
```

Comment:
That's happening if called from the account manger, from the account it will only sync the used one

Reply:
Failed copy paste, thanks